### PR TITLE
configured the tabs

### PR DESCRIPTION
**Pull request by ineskavalenka**

### CODE REVIEW

**Comment by AlexanderSoroka** [comment]
Please, give your pull-requests meaningful names

File: devfirst.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Name=DEV-1
```
Comment (AlexanderSoroka) [inline devfirst.java]: javadoc violation

File: devfirst.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Deadline=21/06/2017 18-00
+ * Prints numbers from 0 to 100. All the numbers that are
+ * multiples of 3 are replaced by 3 * N, where N is the 
+ * required multiplier, so as to obtain a replacement number.
+ */
+
```
Comment (AlexanderSoroka) [inline devfirst.java]: there should not be blank line between comment and class

File: devfirst.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+class TaskDevFirst {
+	public static void main(String[] args) {
+
```
Comment (AlexanderSoroka) [inline devfirst.java]: useless blank line

File: devfirst.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+class TaskDevFirst {
+	public static void main(String[] args) {
+		
+		final int START_COUNT = 0;
+		final int END_COUNT = 100;
+		final int FACTOR = 3;
+		
+		for (int i=START_COUNT;i<=END_COUNT;i++) {
```
Comment (AlexanderSoroka) [inline devfirst.java]: codestyle (spaces)

File: devfirst.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+			if (i % FACTOR == 0) {
+				int temp = i/FACTOR;
+				System.out.println(FACTOR + "*" + temp);
+			} else {
+			System.out.println(i);
```
Comment (AlexanderSoroka) [inline devfirst.java]: indentation is wrong

File: devfirst.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+			}
```
Comment (AlexanderSoroka) [inline devfirst.java]: are spaces/tabs configured correctly?

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: devfirst.java (hunk 1 above)
Comment (ineskavalenka) [reply devfirst.java]: what's wrong?

File: devfirst.java (hunk 5 above)
Comment (ineskavalenka) [inline devfirst.java]: where exactly?

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**